src/content/articles/kubernetes-for-scala-developers-part-one/index.mdx

```diff
@@ -0,0 +1,1052 @@
+    * The Kubelet on the assigned Node pulls container images
+    * Container Runtime creates and starts the actual containers
+    * Status updates flow back through the system
+5) * **Monitoring Cycle** :
+    * The Kubelet monitors container health and restarts containers if needed
+    * The Controllers ensure the desired state is maintained
+
+#### Visualization of the deployment process
+This diagram illustrates how Kubernetes orchestrates deployments through multiple coordinated components, each with
+specific responsibilities in the workflow.
+![Deployment lifecycle](images/kubernetes-deployment-sequence-diagram.png)
+
+:::
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: workload-api
+  labels:
+    demo: k8s-auto-scaling
+spec:
+  selector:
+    matchLabels:
+      type: api
+      language: scala
+      framework: play-framework
+      service: workload
+  minReadySeconds: 1
+  progressDeadlineSeconds: 1200
+  revisionHistoryLimit: 5
+  strategy: # Deployment strategy
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template: # Pod definition
+    metadata:
+      labels:
+        type: api
+        language: scala
+        framework: play-framework
+        service: workload
+    spec:
+      containers:
+        - name: workload-api
+          image: aidocking/pfpw:20.20.31
+          ports:
+            - containerPort: 9000
+          livenessProbe:
+            httpGet:
+              path: /listOfLocations
+              port: 9000
+            failureThreshold: 3
+            timeoutSeconds: 2
+            periodSeconds: 10
+            initialDelaySeconds: 5
+
+          startupProbe:
+            httpGet:
+              path: /listOfLocations
+              port: 9000
+            failureThreshold: 30
+            periodSeconds: 15
+            timeoutSeconds: 2
+
+          readinessProbe:
+            httpGet:
+              path: /listOfLocations
+              port: 9000
+            failureThreshold: 10
+            periodSeconds: 15
+            timeoutSeconds: 10
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: workload-service
+spec:
+  type: ClusterIP
+  selector:
+    type: api
+    language: scala
+    framework: play-framework
+  ports:
+    - port: 9000
+      targetPort: 9000
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: api-workload-ingress
+  annotations:
+    'spec.ingressClassName': "nginx"
+    ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+spec:
+  rules:
+    - host: k8splayworkload.com
+      http:
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+            backend:
+              service:
+                name: workload-service
+                port:
+                  number: 9000
+
+```
+:::
+
+* **minReadySeconds**: let us specify the minimum number of seconds to wait before kubernetes starts considering the pods
```

Comment:
Now this is you. Good!

Keep the bullets' style consistent:
- use backticks for the names instead of bold
- continue with lowercase letter after :
- continue with a noun
- do not use full-stop at the end of the sentence 
- keep the descriptions similar in length

Reply:
But * **item**: is how I'm listing things to explain in the article